src/ui/components/Timeline/Message.js

```diff
@@ -97,7 +102,8 @@ class Message extends React.Component {
       }) >
       timelineMarkerWidth / 2;
 
-    const isHighlighted = highlightedMessage == message.id;
+    const isHighlighted =
```

Comment:
This can be confusing. We probably should add a comment explaining why we need to have a fallback in case `highlightedMessage != message.id`.

Reply:
Also small nit, but we should probably name it `highlightedMessageId` to be more specific?